pkgs/servers/mail/mailman/package.nix

```diff
@@ -55,18 +64,6 @@ buildPythonPackage rec {
       --replace /usr/bin/lynx ${lynx}/bin/lynx
   '';
 
-  # Mailman assumes that those scripts in $out/bin are Python scripts. Wrapping
-  # them in shell code breaks this assumption. Use the wrapped version (see
-  # wrapped.nix) if you need the CLI (rather than the Python library).
-  #
-  # This gives a properly wrapped 'mailman' command plus an interpreter that
-  # has all the necessary search paths to execute unwrapped 'master' and
-  # 'runner' scripts.
-  dontWrapPythonPrograms = true;
```

Comment:
we may need to revert this, but i am curious why unwrapped derivation is default, it would be useful for cli tools to work in situations like `nix run nixpkgs#mailman`

Reply:
Did you test whether the NixOS test still works?

Reply:
I couldnt find ant nixos tests for mailman

Reply:
So, this was never deployed to an actual mailman instance?

Anyways, will try to take a look soonish.

Reply:
That was a typo. There's only a module. I guess we need someone who uses that to make sure it still works.
cc @Ma27 @alyssais @sorpaas @wentasah @petabyteboy

Reply:
I'm pretty sure this isn't going to work.  As the comment says, Mailman tries to execute programs from its configured `bin_dir` (which is set to `"${pkgs.mailmanPackages.mailman}/bin"` by the NixOS module) with a Python interpreter, which would break if it were wrapped.

We used to have a wrapped mailman in `systemPackages` so that running mailman commands directly would work without interfering with the Mailman master's running of its internal commands, but that seems to have got lost at some point.  We can look more into that, but it shouldn't be part of or block this update.

Reply:
Here's what happens, btw:

```
mailman[1257]:   File "/nix/store/kngis48gbfl1a2bdmpv6pm243ngplz7s-python3.10-mailman-3.3.8/bin/master", line 2
mailman[1257]:     PATH=${PATH:+':'$PATH':'}
mailman[1257]:          ^
mailman[1257]: SyntaxError: invalid syntax
```

And [here](https://gitlab.com/mailman/mailman/-/blob/74684ddfbc6a365c7f04e72cc849d13992ccc446/src/mailman/commands/cli_control.py#L120-131) is one of the places in Mailman where it does that.

Reply:
Can't we patch that to remove `sys.executable`?

Reply:
I restored the not wrapping of bin files. I might have a look at this as part of an other MR, as least this fixes currently broken on master `mailman-web`

@dotlambda  I am not sure what you mean about remove `sys.executable` ? I assume we still want the files in `bin/`

Reply:
> I am not sure what you mean about remove `sys.executable` ?

See the link to upstream code @alyssais posted. If upstream called `mailman` or whatever directly instead of `python mailman.py` there wouldn't be a problem.

Reply:
> Can't we patch that to remove `sys.executable`?

Yes, probably. But please not in this PR.

Reply:
yes, i prefer to try this in an other PR, otherwise, i think this one can be merged